ui/qt-lib/src/Ariadne/UI/Qt/Widgets/Dialogs/Send.hs

```diff
@@ -172,19 +227,40 @@ initSend uiWalletFace@UiWalletFace{..} sendInputs = do
         eventType <- QEvent.eventType ev
         dropdownVisible <- QWidget.isVisible dropdownWidget
         if eventType == QEvent.Resize && dropdownVisible
-          then moveDropDown dropdownWidget s $> True
+          then
+            moveDropDown dropdownWidget s $> True
           else return False
 
     for_ checkboxes $
       \AccountCheckbox{..} -> connect_ checkbox QAbstractButton.toggledSignal $ \_ -> accountsSelected as s
 
+  void $ Event.onEvent send $
+    \(ev :: QEvent.QEvent) -> do
+      eventType <- QEvent.eventType ev
+      if eventType == QEvent.Resize
+        then do
+          moveAddReceiverButton s
+          return True
```

Comment:
I'm not sure `return True` is what needed here. We don't consume this event and want parents to do some postprocessing if they need to.
I believe I have some helper in `Util.hs` for the case where every branch returns `False`.
But it'd be nice if you checked docs to confirm my claim :)